docs/application/web/get-started/tv/first-samsung-tv-app.md

```diff
@@ -303,8 +301,7 @@ To run the application on a target device:
 
     3. Confirm that the application launches on the target device.
 
-    > **Note**
-    >
+    > [!NOTE]
```

Comment:
It looks like this note needs a line break.

![image](https://user-images.githubusercontent.com/9958374/71966703-6b25e800-3245-11ea-8ee1-a3abd60726f2.png)

Reply:
Please don't leave a comment here, if an issue is not related to contents but raise an issue on https://github.sec.samsung.net/RS8-TIZENWEBSITE/docs.tizen.org/issues. I will do it this time.

Reference: https://github.sec.samsung.net/RS8-TIZENWEBSITE/docs.tizen.org/issues/246